asylum-connect/catalog-api
# Feature/code coverage

## Description



- Asana ticket:

## PR Checklist




## How to Test

Open Circle CI status check below to see workflow where binary is compiled and e2e tests are run and coverage uploaded to codecov
